docs/newfile.md

```diff
@@ -1,3 +1,21 @@
 # New Markdown File
 
 Texting testing testing
+
+
+1. Thing one
+1. Thing two
+1. Thing three
+
+|Option|Description|
+|---|---|
+|Thing 1|It's nice|
+|Thing 2|It's better|
```

Comment:
This is not better, it's worse

Reply:
Fixed in https://github.com/stuclem/docreviewtest/pull/1/commits/e0012db8e5b1c0d11981a52efa990706c2071807.